packages/neos-ui-redux-store/src/UI/EditModePanel/index.js

```diff
@@ -30,11 +30,9 @@ export const actions = {
 // Export the reducer
 //
 export const reducer = handleActions({
-    [system.INIT]: () => $set(
+    [system.INIT]: payload => $set(
         'ui.editModePanel',
-        new Map({
-            isHidden: true
```

Comment:
btw, can we just throw away fallback state from here and rely on default state defined in Settings? https://github.com/neos/neos-ui/blob/master/Configuration/Settings.yaml#L87